CMakeLists.txt

```diff
@@ -424,12 +425,13 @@ add_subdirectory(${EXTERNAL}/cgltf/tnt)
 add_subdirectory(${EXTERNAL}/xatlas/tnt)
 add_subdirectory(${EXTERNAL}/stb/tnt)
 
+# spirv-tools must come before filamat, as filamat relies on the presence of the
+# spirv-tools_SOURCE_DIR variable.
+add_subdirectory(${EXTERNAL}/spirv-tools)
```

Comment:
Could we not build those libraries when we build for Android? They take a ton of time to compile and we don't need them.

Reply:
Fixed.  Moved these back and moved matdbg so that it lives in the same block as matinfo, which is probably where it should've been anyway!

Btw the debugger will be very useful for Android (imagine connecting to your phone via the web interface) but I will make it opt-in and/or make it build only in the debug configuration.